spec/controllers/branches_controller_spec.rb

```diff
@@ -129,6 +129,23 @@
       end
     end
 
+    context "with extra convergence branche and one non-convergance" do
+      let!(:non_master_branch) { FactoryGirl.create(:branch, :name => 'feature-branch', convergence: false, repository: repository) }
+      let!(:convergence_branch) { FactoryGirl.create(:branch, :name => 'convergence', convergence: true, repository: repository) }
+
+      it "should return two projects" do
+        ## Explicitly load the branches at this point.
+        branch.reload
```

Comment:
the `reload` is not necessary and causes an extra DB call. Just reference the variable to force it to load.